### PR TITLE
Improve phpdoc of model

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by jordisala1991** [comment]
PhpcsFixer is failing

**Comment by VincentLanglet** [comment]
> PhpcsFixer is failing

Fixed

File: src/Model/CollectionInterface.php [hunk 1]
```diff
@@ -21,79 +21,63 @@ interface CollectionInterface
...
-     *
      * @param bool $enabled
      */
     public function setEnabled($enabled);
 
     /**
-     * Get enabled.
-     *
-     * @return bool $enabled
+     * @return bool|null
```
Comment (franmomu) [inline src/Model/CollectionInterface.php:39]:
```suggestion
     * @return bool
```

I guess

File: src/Model/CollectionInterface.php [hunk 2]
```diff
@@ -21,79 +21,63 @@ interface CollectionInterface
...
      */
     public function getEnabled();
 
     /**
-     * Set slug.
-     *
      * @param int $slug
```
Comment (franmomu) [inline src/Model/CollectionInterface.php:44]:
```suggestion
     * @param string $slug
```

File: src/Model/TagInterface.php [hunk 3]
```diff
@@ -21,79 +21,53 @@ interface TagInterface
...
-     *
-     * @return bool $enabled
+     * @return bool
      */
     public function getEnabled();
 
     /**
-     * Set slug.
-     *
      * @param int $slug
```
Comment (franmomu) [inline src/Model/TagInterface.php:44]:
```suggestion
     * @param string $slug
```

**Comment by VincentLanglet** [comment]
Thanks @franmomu fixed

**Review by franmomu — APPROVED** [review]

**Pull request merged**